packages/lambda-send-payment/send-email.js

```diff
@@ -127,15 +126,9 @@ ${displayItems(lineItems, '£', '\r\n')}
 
 Total value: £${amount}
 
-The address you submitted was:
-
-${
-  [(`${firstName} ${lastName}`).trim(), address1, address2, address3, postCode].filter(Boolean).join('\r\n')
-}
-
 Your order will be submitted to the factory at the end of the month.
 
-Once we know the delivery date for your items we'll let you know.
```

Comment:
i know we don't do this right now, but we could start bulk emailing everyone from an order once paria send us the tracking number? That's definitely the nicest flow for the end users, takes the decision making on whether to chase out of their hands. depends on how easy it is to grab everyone's email addresses - do they make it into google sheets so that we can easily and non-technically send out a reminder email?

Reply:
At the moment you'd just collate emails manually from the order folder in the club account.

Reply:
cool, gimme a prod nearer the time and i'll do that when the early june order gets shipped